src/components/App/App.test.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App.component';
+
+it('renders the app', () => {
+  render(<App />);
+  expect(screen.getByText('Welcome to the Challenge!')).toBeInTheDocument();
+});
```

Comment:
You can wrap the test case in with a describe and add a more expressive description to get more information when you run the tests, something like:

```suggestion
describe('App Component', () => {
  it('should render the app', () => {
    render(<App />);
    expect(screen.getByText('Welcome to the Challenge!')).toBeInTheDocument();
  });
});
```